frontend/awx/views/schedules/hooks/useSchedulesFilter.tsx

```diff
@@ -1,26 +1,25 @@
-import { useMemo } from 'react';
-import { IToolbarFilter } from '../../../../../framework';
 import {
-  useDescriptionToolbarFilter,
-  useNameToolbarFilter,
   useCreatedByToolbarFilter,
   useModifiedByToolbarFilter,
 } from '../../../common/awx-toolbar-filters';
+import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
 
 export function useSchedulesFilter() {
-  const nameToolbarFilter = useNameToolbarFilter();
-  const descriptionToolbarFilter = useDescriptionToolbarFilter();
   const createdByToolbarFilter = useCreatedByToolbarFilter();
   const modifiedByToolbarFilter = useModifiedByToolbarFilter();
 
-  const toolbarFilters = useMemo<IToolbarFilter[]>(
-    () => [
-      nameToolbarFilter,
-      descriptionToolbarFilter,
-      createdByToolbarFilter,
-      modifiedByToolbarFilter,
-    ],
-    [nameToolbarFilter, descriptionToolbarFilter, createdByToolbarFilter, modifiedByToolbarFilter]
-  );
+  const toolbarFilters = useDynamicToolbarFilters({
```

Comment:
Looks like the inventory filter is not loading options- might need to be added/edited in `knownAwxFilterKeys`
![Screenshot 2024-04-17 at 1 33 54 PM](https://github.com/ansible/ansible-ui/assets/89094075/e058b30c-025e-436b-bb16-9220f3cb5f16)